src/api/XRSession.js

```diff
@@ -121,6 +123,38 @@ export default class XRSession extends EventTarget {
     };
     polyfill.addEventListener('@@webxr-polyfill/vr-present-start', this[PRIVATE].onPresentationStart);
 
+    this[PRIVATE].onSelectStart = evt => {
+      // Ignore if this event is not for this session.
+      if (evt.sessionId !== this[PRIVATE].id) {
+        return;
+      }
+
+      this.dispatchEvent('selectstart', {
+        frame: this[PRIVATE].frame,
+        inputSource: evt.inputSource
+      });
+    };
+    polyfill.addEventListener('@@webxr-polyfill/input-select-start', this[PRIVATE].onSelectStart);
+
+    this[PRIVATE].onSelectEnd = evt => {
+      // Ignore if this event is not for this session.
+      if (evt.sessionId !== this[PRIVATE].id) {
+        return;
+      }
+
+      this.dispatchEvent('selectend', {
+        frame: this[PRIVATE].frame,
+        inputSource: evt.inputSource
+      });
+
+      // Sadly, there's no way to make this a user gesture.
+      this.dispatchEvent('select',  {
+        frame: this[PRIVATE].frame,
+        inputSource: evt.inputSource
+      });
+    };
+    polyfill.addEventListener('@@webxr-polyfill/input-select-end', this[PRIVATE].onSelectEnd);
+
     this.onblur = undefined;
     this.onfocus = undefined;
     this.onresetpose = undefined;
```

Comment:
Need to add a few `on` handlers for select, selectstart, selectend (not yet actually implemented!)